main/src/ca/uwaterloo/flix/language/phase/HtmlDocumentor.scala

```diff
@@ -407,9 +407,9 @@ object HtmlDocumentor {
     * i.e. this should be called before `pairModules`.
     */
   private def filterEffect(eff: Effect): Effect = eff match {
-    case Effect(eff, parent, _) =>
+    case Effect(e, parent, _) =>
```

Comment:
arent these usually called `sym` something?

Reply:
This is an actual effect AST node, not just a sym.